masmini-common/src/main/java/masmini/Resource.kt

```diff
@@ -34,80 +32,37 @@ open class Resource<out T> @PublishedApi internal constructor(val value: Any?) {
      * Get the current value if successful, or null for other cases.
      */
     fun getOrNull(): T? =
-        when {
-            isLoading -> (value as Loading<T>).value
-            isSuccess -> value as T?
-            else -> null
-        }
```

Comment:
Restore this, we may want to recover the value that we store in the resource if loading (for example, download percentages)